dev/customer_testing/ci.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+# This is called from .cirrus.yml and the LUCI recipes:
+# https://flutter.googlesource.com/recipes/+/refs/heads/master/recipe_modules/adhoc_validation/resources/customer_testing.sh
+
+# This script does not assume that "flutter update-packages" has been
+# run, to allow CIs to save time by skipping that steps since it's
+# largely not needed to run the flutter/tests tests.
+#
+# However, we do need to update this directory and the tools directory.
+pub get
+(cd ../tools; pub get) # used for find_commit.dart below
+
+# Next we need to update the flutter/tests checkout.
+#
+# We use find_commit.dart so that we pull the version of flutter/tests
+# that was contemporary when the branch we are on was created. That
+# way, we can still run the tests on long-lived branches without being
+# affected by breaking changes on trunk causing changes to the tests
+# that wouldn't work on the long-lived branch.
+#
+# (This also prevents trunk from suddenly failing when tests are
+# revved on flutter/tests -- if you rerun a passing customer_tests
+# shard, it should still pass, even if we rolled one of the tests.)
+rm -rf ../../bin/cache/pkg/tests
+git clone https://github.com/flutter/tests.git ../../bin/cache/pkg/tests
```

Comment:
We'll also need a windows version of this file (or one Dart file)

Reply:
I have no way to test it that I can think of but I tried to create one. Let me know if it works? :-)

Reply:
It passed - https://chromium-swarm.appspot.com/task?id=547cffaf10ff2710